### ISSUE TITLE
[Feature Request]: Instructions for running locally

### ISSUE THREAD

**Issue by poema-cavalcanti** [ShootMe/FallGuysStats#238]
### What is your request?

It would be nice to have instructions on what all the install and run instructions are since the tracker is now getting blocked by anti-virus software when downloading the exe. This would also make it easier to stay updated and use workarounds since we can just pull master or another branch in a pinch.

**Comment by qutrits** [comment]
I can't think of any other way other than adding [this guide](https://github.com/ShootMe/FallGuysStats#first-run).
